.github/workflows/release.yml

```diff
@@ -5,6 +5,10 @@ on:
     tags:
       - "v*"
 
+permissions:
+  contents: write
```

Comment:
Why write ? Neither https://docs.npmjs.com/generating-provenance-statements nor https://docs.npmjs.com/generating-provenance-statements#publishing-packages-with-provenance-via-github-actions, mention `contents` having write permissions, only id-token.

Reply:
This is because auf the releaser script so kt can create a release